setup.cfg

```diff
@@ -72,16 +72,16 @@ install_requires =
     # Docker
     container-inspector==33.0.0
     # ScanCode-toolkit
-    scancode-toolkit[packages]==32.3.0
+    scancode-toolkit[packages] @ git+https://github.com/nexB/scancode-toolkit.git@c9c023e7712044475578d03814f34bb03435d949
```

Comment:
Not great. Could you push a release instead?

Reply:
Done! Updated to https://github.com/aboutcode-org/scancode-toolkit/releases/tag/v32.3.2